studio/components/interfaces/Auth/RateLimits/RateLimits.tsx

```diff
@@ -0,0 +1,336 @@
+                              <Link passHref href={`/project/${projectRef}/auth/providers`}>
+                                <Button asChild type="default" className="mt-2">
+                                  <a>View providers configuration</a>
+                                </Button>
+                              </Link>
+                            </AlertDescription_Shadcn_>
+                          </Alert_Shadcn_>
+                        )}
+                      </FormItem_Shadcn_>
+                    )}
+                  />
+                </FormSectionContent>
+              </FormSection>
+
+              <FormSection
+                id="token-refresh"
+                header={
+                  <FormSectionLabel
+                    description={
+                      <p className="text-foreground-light text-sm">
+                        How many sessions can be refreshed in a 5 minute interval
+                      </p>
+                    }
+                  >
+                    Rate limit for token refreshes
+                  </FormSectionLabel>
+                }
+              >
+                <FormSectionContent loading={false}>
+                  <FormField_Shadcn_
+                    control={form.control}
+                    name="RATE_LIMIT_TOKEN_REFRESH"
+                    render={({ field }) => (
+                      <FormItem_Shadcn_>
+                        <FormControl_Shadcn_>
+                          <Input_Shadcn_ disabled type="number" {...field} />
+                        </FormControl_Shadcn_>
+                        {field.value > 0 && (
+                          <p className="text-foreground-lighter text-sm">
+                            This is equivalent to {field.value * 12} requests per hour
+                          </p>
+                        )}
+                        <FormMessage_Shadcn_ />
+                      </FormItem_Shadcn_>
+                    )}
+                  />
+                </FormSectionContent>
+              </FormSection>
+
+              <FormSection
+                id="verify"
+                header={
+                  <FormSectionLabel
+                    description={
+                      <p className="text-foreground-light text-sm">
+                        How many OTP/Magic link verifications can be made in a 5 minute interval
+                      </p>
+                    }
+                  >
+                    Rate limit for token verifications
+                  </FormSectionLabel>
+                }
+              >
+                <FormSectionContent loading={false}>
+                  <FormField_Shadcn_
+                    control={form.control}
+                    name="RATE_LIMIT_VERIFY"
+                    render={({ field }) => (
+                      <FormItem_Shadcn_>
+                        <FormControl_Shadcn_>
+                          <Input_Shadcn_ disabled type="number" {...field} />
```

Comment:
Same with this one?

Reply:
https://github.com/supabase/supabase/pull/18265#discussion_r1363577819